cockroachcloud/serverless-faqs.md

```diff
@@ -168,18 +168,10 @@ $ cockroach sql \
-
-{{site.data.alerts.callout_info}}
-Creating a [changefeed](../{{site.current_cloud_version}}/create-and-configure-changefeeds.html) for a {{ site.data.products.serverless }} cluster without first entering billing information will cause the following error: `pq: Outbound IO is disabled by configuration`.
-{{site.data.alerts.end}}
+You can run a "sinkless" changefeed to the current SQL session with [`EXPERIMENTAL CHANGEFEED FOR`](../{{site.current_cloud_version}}/changefeed-for.html) or [`CREATE CHANGEFEED`](../{{site.current_cloud_version}}/create-changefeed.html), or you can [run a changefeed to a configurable sink](../{{site.current_cloud_version}}/changefeed-sinks.html).
 
 ### Can I backup my {{ site.data.products.serverless }} cluster? Does Cockroach Labs take backups of my cluster?
 
-The [**Backups** page](backups-page.html) allows you to restore your cluster from automatic full cluster backups, which are performed hourly and stored for 30 days. {{ site.data.products.db }} does not take incremental backups of {{ site.data.products.serverless }} clusters, or allow database or table level restores from automatic full cluster backups. However, you can also backup and restore your {{ site.data.products.serverless }} cluster manually. If you don't have [billing information on file](billing-management.html) for your organization, you can [take backups locally](run-bulk-operations.html#backup-and-restore-data) to `userfile`. Once you enter billing information, even if you leave your spend limit at the $0 default, you can also [backup to cloud storage](run-bulk-operations.html#backup-and-restore-data).
-
-{{site.data.alerts.callout_info}}
-Running a [bulk operation](run-bulk-operations.html) to cloud storage from a {{ site.data.products.serverless }} cluster without first entering billing information will cause the following error: `external network access is disabled`.
-{{site.data.alerts.end}}
+The [**Backups** page](backups-page.html) allows you to restore your cluster from automatic full cluster backups, which are performed hourly and stored for 30 days. {{ site.data.products.db }} does not take incremental backups of {{ site.data.products.serverless }} clusters, or allow database or table level restores from automatic full cluster backups. However, you can also backup and restore your {{ site.data.products.serverless }} cluster manually. You can [take backups locally](run-bulk-operations.html#backup-and-restore-data) to `userfile` or [backup to cloud storage](run-bulk-operations.html#backup-and-restore-data).
```

Comment:
The mention of 'restore' could link to https://www.cockroachlabs.com/docs/cockroachcloud/backups-page.html

The mention of 'full cluster backups' could link to https://www.cockroachlabs.com/docs/v22.1/take-full-and-incremental-backups#full-backups

The mention of `userfile` could be a link to https://www.cockroachlabs.com/docs/stable/use-userfile-for-bulk-operations.html